libs/contact/state/src/contact.module.ts

```diff
@@ -0,0 +1,8 @@
+import { NgModule } from '@angular/core';
+
+@NgModule({
+	declarations: [],
+	imports: [],
+	providers: [],
+})
+export class DaffContactModule {}
```

Comment:
This should export/inject the effects and reducers, and should be called `DaffContactStateModule`

Reply:
Fixed in 061de9d8ecfc2da4c3df5910d597a21954e70e76

Reply:
The name on the module should be changed to `DaffContactStateModule`. And the effect should be provided here unless there's a reason not to

Reply:
Sorry for missing this comment awhile ago. Fixed in 3accd4e2f658fa0ce8269bc5fbf7e5367e610e65.